company-profiles/3blocks.md

```diff
@@ -0,0 +1,29 @@
+# 3Blocks
+
+## Company blurb
+
+3Blocks is a tokenized fractional real estate ownership platform. We bring lucrative American real estate on the blockchain and enable people around the world to invest in American real estate from the comfort of their living rooms.
+
+## Company size
+
+5-10 employees.
+
+## Remote status
+
+As a remote-friendly company, we prioritize flexibility, enabling our employees to achieve work-life balance while delivering exceptional results from the comfort of their preferred locations.
+
+## Region
+
+Worldwide
+
+## Company technologies
+
+Coupled with an advanced tech stack encompassing ReactJS, NestJS, Node.js, AWS, and Blockchain, our company leverages cutting-edge technologies to deliver innovative and scalable solutions, ensuring unparalleled value to our clients and users.
+
+## Office locations
+
+4910 E Adamo Dr, Unit B, Tampa, FL - 33605, United States.
+
+## How to apply
+
+Our open positions are listed on our [Linkedin](https://www.linkedin.com/company/3blocks-io/) page.
```

Comment:
```suggestion
Our open positions are listed on our [Linkedin](https://www.linkedin.com/company/3blocks-io/jobs/) page.
```

Reply:
@KieranRobson updated the above LinkedIn jobs url